src/winetricks

```diff
@@ -1076,10 +1076,10 @@ w_verify_sha256sum()
             w_warn "sha256sum mismatch! However --force was used, so trying anyway. Caveat emptor."
         else
             case ${LANG} in
-                pl*) w_die "Niezgodność sumy sha256sum! Zmień nazwę ${_W_vs_file} i spróbuj ponownie." ;;
-                pt*) w_die "sha256sum não confere! Renomeie ${_W_vs_file} e tente novamente. Alternativamente, use --force para ignorar esta verificação." ;;
-                ru*) w_die "Контрольная сумма sha256sum не совпадает! Переименуйте файл ${_W_vs_file} и попробуйте еще раз." ;;
-                *) w_die "sha256sum mismatch! Rename ${_W_vs_file} and try again. Alternatively, use --force to ignore this check." ;;
+                pl*) w_askpermission "Niezgodność SHA256!\n\nURL: ${_W_url}\nPobrano: ${_W_gotsha256sum}\nOczekiwany: ${_W_vs_wantsum}\n\nCzęsto jest to wynikiem zaktualizowanego pakietu, takiego jak vcrun2019.\nJeśli chcesz zaakceptować ryzyko, możesz pominąć tę kontrolę.\n\nKontynuować mimo wszystko?" ;;
+                pt*) w_askpermission "SHA256 incorreto!\n\nURL: ${_W_url}\nBaixado: ${_W_gotsha256sum}\nEsperado: ${_W_vs_wantsum}\n\nIsso geralmente é o resultado de um pacote atualizado, como vcrun2019.\nSe você estiver disposto a aceitar o risco, poderá ignorar essa verificação.\n\nContinue de qualquer maneira?" ;;
+                ru*) w_askpermission "Несоответствие SHA256!\n\nURL: ${_W_url}\nСкачано: ${_W_gotsha256sum}\nОжидал: ${_W_vs_wantsum}\n\nЧасто это результат обновленного пакета, такого как vcrun2019.\nЕсли вы готовы принять риск, вы можете обойти эту проверку.\n\nПродолжайте в любом случае?" ;;
+                *) w_askpermission "SHA256 mismatch!\n\nURL: ${_W_url}\nDownloaded: ${_W_gotsha256sum}\nExpected: ${_W_vs_wantsum}\n\nThis is often the result of an updated package such as vcrun2019.\nIf you are willing to accept the risk, you can bypass this check.\n\nContinue anyway?" ;;
```

Comment:
The message should preserve the suggestion to use `--force` to avoid the prompt.

Reply:
Added.

Please let me know if the dialog is too verbose.